js-pkg/tests/package.json

```diff
@@ -10,7 +10,7 @@
   "license": "ISC",
   "dependencies": {
     "bun-types": "^0.5.7",
-    "driftdb": "1.0.10"
+    "driftdb": "*"
```

Comment:
Does this run the tests against the driftdb that's been published to npm instead of the local one?

Reply:
the opposite; it runs tests against the version in the repo

Reply:
well, I guess it depends what's in `package-lock.json`? but in this case it's running against the version in the repo I think

Reply:
Interesting - it must be that npm workspaces overrides this and makes it point to the local version?

Reply:
Or maybe it only overrides it when the version matches the one in the local repo 🤔 

I just tried making local changes, and they showed up in the test output. Then I changed the driftdb version to some old version and reran `npm i` from the root of the repo and it looked like it was running the old driftdb version, not the local one.

Anyway, was just curious